sdcm/fill_db_data.py

```diff
@@ -1008,7 +1008,7 @@ class FillDatabaseData(ClusterTester):
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 2,
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 3,
                         "SELECT v1, v2 FROM range_tombstones_test WHERE k = %d" % 4,
-                        "#REMOTER_RUN sudo nodetool flush",
```

Comment:
why those are in the same commit ?

Reply:
they were missing from previous related commit, and once the test really started running (apart from all the hassle to have 2019.1 jobs running), i figured out that it was missing.

BTW, once 2019.1.22 will be released, we will revert this (and the previous one)

Reply:
not sure we have a separate branch for the upgrades anyow